Makefile

```diff
@@ -24,7 +24,7 @@ doc:
 
 tests:
 	docker run --name ld-test-redis -p 6379:6379 -d redis
-	@$(REBAR3) ct --dir="test,test-redis" --logdir logs/ct --cover
+	@$(REBAR3) ct --dir="test,test-redis" --logdir logs/ct
```

Comment:
First problem is that it doesn't work with the cover arguments in current OTP 27.